src/PerfView/JitStats.cs

```diff
@@ -160,6 +160,7 @@ public static void ToHtml(TextWriter writer, TraceProcess stats, TraceLoadedDotN
                     "<TH Title=\"The number of times the JIT was invoked for methods in this module\">Num Compilations</TH>" +
                     "<TH Title=\"The total amount of IL processed by the JIT for all methods in this module\">IL Size</TH>" +
                     "<TH Title=\"The total amount of native code produced by the JIT for all methods in this module\">Native Size</TH>" +
+                    "<TH Title=\"The total amount of heap memory allocated for the code produced by the JIT for all methods in this module\">Allocated Heap Size</TH>" +
```

Comment:
For most people, these events aren't going to be collected, as they're too expensive to turn on by default.  I would recommend that you make the addition of this column conditional on having the data.

Reply:
Sure. What is the right way to check that? Just don't display all of this if `TotalAllocSize == 0` or have some flag somewhere in TraceManagerProcess.cs inside `LoaderHeapAllocRequest` delegate?

Reply:
Yes, using `TotalAllocSize==0` is a good way to do it.

Reply:
There is lot of conditional formatting code that needs to be turned off if we need to display this column conditionally. E.g. Adding the format for [extra column in moduleTableRow](https://github.com/microsoft/perfview/pull/1289/files#diff-02da9e6f69caee6ee8d847ae7dbceb6f9dde9c94428bf278c6cc056ab786fec7R175). I am thinking of displaying `N/A` or `-` if there is no data instead of `0`? In the help section, I will add a line saying how can someone view that data. Thoughts?

Reply:
Alternatively, I will need to just add this as a last column which will be easier to turn on and off.

Reply:
I think `N/A` would probably be OK.  The key is to make sure it's clear that the data isn't there (as opposed to 0).